charts/cluster-proportional-autoscaler/values.yaml

```diff
@@ -0,0 +1,75 @@
+affinity: {}
+config: {}
+#  ladder:
+#    coresToReplicas:
+#      - [ 1, 1 ]
+#      - [ 64, 3 ]
+#      - [ 512, 5 ]
+#      - [ 1024, 7 ]
+#      - [ 2048, 10 ]
+#      - [ 4096, 15 ]
+#    nodesToReplicas:
+#      - [ 1, 1 ]
+#      - [ 2, 2 ]
+#  linear:
+#    coresPerReplica: 2
+#    nodesPerReplica: 1
+#    min: 1
+#    max: 100
+#    preventSinglePointFailure: true
+#    includeUnschedulableNodes: true
+image:
+  repository: k8s.gcr.io/cpa/cluster-proportional-autoscaler-amd64
```

Comment:
Could you update this to use the multi-arch image - `k8s.gcr.io/cpa/cluster-proportional-autoscaler`

Reply:
Done